src/main/java/org/gelecekbilimde/scienceplatform/auth/model/request/RegisterRequest.java

```diff
@@ -20,30 +20,30 @@
 public class RegisterRequest {
 
 	@NotNull(message = "firstname cannot be null")
-	@Size(max = 25, message = "firstname should not be less than 25")
+	@Size(max = 25, min = 2 , message = "firstname should be more than 2 and less than 25")
 	private String firstname;
 
 	@NotNull(message = "lastname cannot be null")
-	@Size(max = 25, message = "lastname should not be less than 25")
+	@Size(max = 25, min = 2, message = "lastname should be more than 2 and less than 25")
 	private String lastname;
 
 	@NotNull(message = "email cannot be null")
 	@Email(message = "email should be valid")
-	@Size(max = 255, message = "email should not be less than 255")
+	@Size(max = 255, message = "email should be less than 255")
 	private String email;
 
 	@NotNull(message = "password cannot be null")
-	@Size(max = 255, message = "password should not be less than 255")
+	@Size(max = 255, min = 8, message = "password should be more than 8 and less than 255")
 	private String password;
 	private String biography;
 
 	@JsonFormat(pattern = "yyyy-MM-dd")
 	private LocalDate birthDate;
 
-	@Size(max = 255, message = "gender should not be less than 255")
+	@Size(max = 255, message = "gender should be less than 255")
 	private String gender;
 
-	@Size(max = 255, message = "degree should not be less than 255")
+	@Size(max = 255, message = "degree should be less than 255")
```

Comment:
Message yazmamız şart mı? Spring'in exception'larını handle edip yazabiliriz. Ek olarak önce minimum değeri sonra maksimum değeri eklemeliyiz.
```suggestion
	@Size(min = 2, max = 255)
```